packages/compiler/src/server/serverlib.ts

```diff
@@ -114,7 +114,7 @@ export function createServer(host: ServerHost): Server {
   });
   const compilerHost = createCompilerHost();
   const npmPackageProvider = new NpmPackageProvider(compilerHost);
-  const emitterProvider = new EmitterProvider(npmPackageProvider);
+  const libProvider = new LibraryProvider(npmPackageProvider);
```

Comment:
you dont need libProvidre, instead you can create emitterProvider and linterProvider which is instance of libProvider